CHANGELOG.md

```diff
@@ -146,6 +146,7 @@ This means that druid no longer requires cairo on macOS and uses Core Graphics i
 - GTK: Support disabled menu items. ([#897] by [@jneem])
 - GTK: Support file filters in open/save dialogs. ([#903] by [@jneem])
 - GTK: Support DPI values other than 96. ([#904] by [@xStrom])
+- GTK: Don't crash when receiving an external command while a file dialog is visible. ([#1043] by [@jneem])
```

Comment:
We're no longer accepting changes to 0.6.0. :smirk:

Reply:
I was wondering why the other PR numbers seemed so low...